tools/operator/paths/paths.go

```diff
@@ -57,6 +57,18 @@ func TemplatesDir() string {
 	return componentPath(currentDir, "templates")
 }
 
+//ScriptsDir --
+func ScriptsDir() string {
+	currentDir, err := GetCurrentDir()
+	if err != nil {
+		logger.ERROR("ScriptsDir function is failed in getting current directory")
```

Comment:
grammar:
    ScriptsDir function failed to get current directory

Reply:
Done